### ISSUE TITLE
Could not find upload info for artboard

### ISSUE THREAD

**Issue by buveur** [avocode/avocode#1679]
* **Version**: 2.16.1
* **Sketch plugin version**: 47.1.0

* **Web/Desktop**: Desktop

* **Platform**: darwin

 
* **Avocode UID**: 59781
---

Can't upload anything into project named "unisender.com" It works just fine with other projects.
https://i.gyazo.com/39811b6416275933e06fb87c5e592cf4.gif
```
Error: Could not find upload info for artboard CF5C0CBA-27C2-428F-A2E2-13E4D54BE74D
    at file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:3383162
    at List.__iterate (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:2208:13)
    at List.forEach (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:4383:19)
    at t._handleUserDataDesignsRequest (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:3383049)
    at emitTwo (events.js:106:13)
    at t.emit (events.js:194:7)
    at t.value (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:1147950)
    at file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:1510922
    at List.__iterate (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:2208:13)
    at List.forEach (/Applications/
```

**Comment by jakubzitny** [comment]
Hi @buveur, thanks for reporting this 🙂 This is a bug and we'll be fixing it for the next release.

**Comment by jakubzitny** [comment]
Hi, this should be fixed in latest Avocode `2.16.2`. Please update and let us know if you find any other problems :wink:
